src/main/java/internet/store/model/Order.java

```diff
@@ -0,0 +1,37 @@
+package internet.store.model;
+
+import java.util.List;
+
+public class Order {
+    private Long orderId;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(Long userId) {
+        this.userId = userId;
+    }
```

Comment:
```suggestion
    public Order(Long userId) {
        this.userId = userId;
        products = new ArrayList<>():
    }
```

Reply:
same in shoppingCart

Reply:
I hope I get it now - as soon as user is created, we set (create) empty product list so it can add products into Shopping cart and/or order. Right?